### PR TITLE
Dualspark master

### PR DESCRIPTION
**Pull request by sesas**
@gimballock Do you want to merge this in?

### CODE REVIEW

File: src/examples/basic.py [hunk 1]
```diff
@@ -9,14 +9,15 @@ class MyEnvClass(NetworkBase):
     """
 
     def create_hook(self):
-        # Add a bastion host as a child stack in the environment
-        self.add_child_template(Bastion())
+        pass
+        # # Add a bastion host as a child stack in the environment
```
Comment (odandia) [inline src/examples/basic.py:13]: I still don't think you should remove all the content from the basic example, that really defeats the purpose of the example

**Comment by odandia** [comment]
Please update [version.py](https://github.com/DualSpark/cloudformation-environmentbase/blob/master/src/environmentbase/version.py) with any pushes to master, along with a new version tag

File: src/examples/basic.py (hunk 1 above)
Comment (sesas) [inline src/examples/basic.py:13]: @gimballock what do you say to this ^ comment?  Does the basic example in master work as it is? or do we need to modify it to make it work?  Or else, why did you add that commit to this PR?

**Comment by gimballock** [comment]
These changes are no longer relevant given the recent changes to use Fn::GetAZs instead of RegionMap.

**Comment by sesas** [comment]
You should include a link when referring to changes that are not obvious (for posterity as well)
